doc/plugin_agent_svidstore_aws_secretsmanager.md

```diff
@@ -18,6 +18,49 @@ The format that is used to store in a secret the issued identity is the followin
 }
 ```
 
+### Required AWS IAM permissions
```

Comment:
what do you think about follow the same structure used on [gcp plugin](https://github.com/spiffe/spire/blob/main/doc/plugin_agent_svidstore_gcp_secretmanager.md#required-gcp-permissions)?  

And maybe avoid adding an example, so we don't have to worry about keeping it updated?

Reply:
Yes good point, I'll remove the example.